packages/desktop-client/src/components/Notifications.tsx

```diff
@@ -164,17 +172,21 @@ function Notification({
               style={{
                 backgroundColor: 'transparent',
                 border: `1px solid ${
-                  positive ? colors.g5 : error ? colors.r4 : colors.y3
+                  positive
+                    ? theme.noticeAccent
+                    : error
+                    ? theme.errorAccent
+                    : theme.warningAccent
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏same

Reply:
This is a border within the same function as the above.
Line 129 border = g5/r5/y4 
then 50 lines down (same function still on borders) on line 175 we use g5/r4/y3
What's the point of changing the border color within the same function here? Shouldn't these be the same?